build.gradle

```diff
@@ -100,6 +100,19 @@ allprojects {
 
 final boolean isTravis = System.env.TRAVIS == 'true'
 
+/**
+ * Defines whether the given {@code project} should expose its JavaDoc to spine.io Web site.
```

Comment:
I think, it's `Determines`, `Javadoc` and `website`.

Let's have `Javadoc` spelling through this changeset.